DP1.10081.002/Data.quality.txt

```diff
@@ -0,0 +1,3 @@
+Community taxonomy data are derived from marker gene sequences, which must pass basic data quality assurance/quality control (QA/QC) checks, including meeting a minimum sequencing depth (e.g. number of sequences per sample), a maximum number of ambiguous base calls, and a minimum quality threshold. The specifics of the data quality criteria used may change over time as new methods become available and standards change. Within the expanded download package, the table mct_taxonTableLabSummary includes details on methods and code used by the external facility during sample processing. 
```

Comment:
Are there specific fields you can point to that people should use to evaluate the quality of a given sequence/sample/record? Are we reporting confidence estimates for the taxon assignments? In general in the QSG data quality section we try to focus on what information within the data can be used to evaluate quality.